source/server/http/admin.cc

```diff
@@ -653,7 +688,18 @@ void AdminFilter::onComplete() {
   Buffer::OwnedImpl response;
   Http::HeaderMapPtr header_map{new Http::HeaderMapImpl};
   RELEASE_ASSERT(request_headers_);
-  Http::Code code = parent_.runCallback(path, *request_headers_, *header_map, response);
+  Http::Code code;
+  bool end_stream = true;
+
+  if (path.find("/hystrix_event_stream") == std::string::npos) {
+    handler_info_ = std::make_unique<HandlerInfoImpl>();
+    code = parent_.runCallback(path, *request_headers_, *header_map, response, *handler_info_);
+  } else {
+    handler_info_ =
+        std::make_unique<Extensions::StatSinks::Common::HystrixHandlerInfoImpl>(callbacks_);
+    code = parent_.runCallback(path, *request_headers_, *header_map, response, *handler_info_);
+    end_stream = false;
```

Comment:
If we decide to remove `handler_info_`, could we remove the rest of this special case?  Could we just allow the callback to set the end_stream (it could still be defaulted to true so we don't need to change the other handlers)?

Reply:
How about, instead of involving the handlers and adding more arguments to their interface, add a `bool streaming_` to `struct UrlHandler` and let `AdminImpl::runCallback` handle the `end_stream` value?